extensions/ql-vscode/package.json

```diff
@@ -346,6 +346,12 @@
           "default": "",
           "patternErrorMessage": "Please enter a valid folder",
           "markdownDescription": "The name of the folder where we want to create queries and query packs via the \"CodeQL: Create Query\" command. The folder should exist."
+        },
+        "codeQL.autogenerateQlPacks": {
+          "type": "string",
+          "default": "",
+          "patternErrorMessage": "Please choose between 'Yes', 'No' and 'No, and never ask me again'",
+          "markdownDescription": "The choice of whether to autogenerate QL packs for the current workspace. The choice is 'Yes', 'No' or 'No, and never ask me again'."
```

Comment:
Just to check my understanding: is `codeQL.autogenerateQlPacks` ever "Yes" or "No"? 🤔 It looks like `codeQL.autogenerateQlPacks` is either not set, or it's equal to "No, and never ask me again". 

If that's correct, would this make more sense as a binary true/false setting? (e.g. `enableAutogeneratedQlPacks`: `true`/`false`)

Reply:
Good point. Two of the options are indeed redundant for this setting. If we choose:
- "yes" -> we won't store the config option
- "no" -> we won't store the config option
- "never" -> we store the config option

You could also argue that we only need one option here, but I'd like to keep it this way to mirror the options we present to the user.